app/models/rating.rb

```diff
@@ -32,8 +32,8 @@ def self.user_allowed_to_rate?(job:, user:)
 
   def self.average_score(round: nil)
     score = average(:score)
-    score = score.round(round) if !score.nil? && round
-    score
+    score = score.round(round) if score.present? && round
+    score.to_f
```

Comment:
This will return `0.0` if `score` is `nil`. Add a guard clause for `score.nil?` and return  or early if truthy.